Makefile

```diff
@@ -766,6 +773,14 @@ testlogic: TESTFLAGS := -test.v $(if $(FILES),-show-sql)
 testlogic: bin/logictest.test
 	cd pkg/sql/logictest && logictest.test -test.run "$(TESTS)" -test.timeout $(TESTTIMEOUT) $(TESTFLAGS)
 
+# Run make testccllogic to run all of the CCL logic tests. Specify test files to run
+# with make testccllogic FILES="foo bar".
+testccllogic: ## Run SQL Logic Tests.
+testccllogic: TESTS := $(if $(FILES),TestLogic$$//^$(subst $(space),$$|^,$(FILES))$$$(if $(SUBTESTS),/$(SUBTESTS)),TestLogic)
+testccllogic: TESTFLAGS := -test.v $(if $(FILES),-show-sql)
```

Comment:
Ditto here. Can you share at least these two recipe-specific variables with `testlogic`?

```make
testlogic testccllogic: TESTS := $(if $(FILES),TestLogic$$//^$(subst $(space),$$|^,$(FILES))$$$(if $(SUBTESTS),/$(SUBTESTS)),TestLogic)
testlogic testccllogic: TESTFLAGS := -test.v $(if $(FILES),-show-sql)
```

Reply:
Done.